components/PageLayout.tsx

```diff
@@ -14,7 +12,7 @@ interface Props {
 }
 
 const PageLayout: React.FunctionComponent<Props> = props => (
-    <Layout contentTitle={props.contentTitle} metaTags={props.metaTags}>
+    <Layout title={props.title} contentTitle={props.contentTitle} metaTags={props.metaTags}>
```

Comment:
If we flip these in `Article`, then we could do something like this in this file:
```
    const title = props.alternateTitle
      ? props.alternateTitle
      : props.title
      ? `${props.title} - ${siteTitle}`
      : siteTitle
  ```
  which would achieve the same goal. 
  
  We don't seem to have a rule about nested ternaries I don't think? I kind of like them if they're relatively easy to read.

Reply:
This seems reasonable to me